client/db/bolt/db_test.go

```diff
@@ -211,6 +212,21 @@ func TestAccounts(t *testing.T) {
 	}
 }
 
+// TODO: Noticed something weird with TestWallets, when I have
+// TestAccountsBackupAndRestore before it it fails with output:
+//
+// --- FAIL: TestWallets (0.36s)
+//     db_test.go:312: 323 milliseconds to insert 50 Wallet
+//     db_test.go:343: update time can't be right: 2020-03-19 13:56:14 +0000 GMT < 268418715551-05-04 18:09:57 +0000 GMT
```

Comment:
I think the issue is with `RandomWallet` setting a random `BalUpdate` rather than the current time.  `TestWallet` was just getting lucky with the random number choosing a time in the past, but moving the order breaks it.

Reply:
There is also a precision loss issue there as well I think
```
--- FAIL: TestWallets (0.02s)
    db_test.go:321: 3 milliseconds to insert 1 Wallet
    dbtest.go:227: BalUpdate mismatch. 2020-03-19 23:47:08 +0000 GMT != 2020-03-19 23:47:08.621776051 +0000 GMT m=+1.280006788
```
Notice the `.621776051` throwing off the comparison between the restored from db entity and the original in-memory entity. Since `BalUpdate` when persisted is the `Unix()` equivalent of it losses some precision as a result. I don't think the added precision is needed. Will be putting up a PR to rectify that.

Reply:
Curious to know your thoughts on the precision issue @chappjc, using the current time in `RandomWallet` fixes the issue but I think what I noticed still stands. My plan was to make `BalUpdate` an `int64` for storing unix time in seconds, this way it stays consistent for in-memory types and retrieved values from db. Thoughts?

Reply:
For the sake of the DEX, any precision below millisecond should just be dropped, and I think we are doing that where it matters. If the tests are failing because of sub-millisecond precision, maybe just fix the tests.